docs/fieldnorm/verification_validation.md

```diff
@@ -1,7 +1,70 @@
+The following citation is taken from IMDRF/SaMD WG/N23 FINAL:2015 "Software as a Medical Device (SaMD): Application of Quality Management System "
 
-Donec id nibh aliquam, euismod velit id, aliquam ante. Vestibulum finibus interdum tellus quis efficitur. Quisque convallis nec nisl ut sollicitudin. Sed fringilla tellus et massa viverra interdum. Curabitur egestas diam at dui consectetur, et dapibus dui dignissim. Maecenas eget pulvinar enim. Etiam sodales id dui id venenatis. Duis id pellentesque tellus. Quisque enim justo, tristique id volutpat in, euismod ac tellus. Etiam faucibus velit sed purus commodo, eu vehicula velit egestas. Vivamus faucibus est et tortor imperdiet convallis. Vestibulum nulla enim, blandit a purus eget, cursus pellentesque lectus. Nulla finibus mi euismod eros lobortis, vitae eleifend erat rutrum.
\ No newline at end of file
+> Verification and Validation
+> The verification and validation (V&V) activities should be targeted towards the criticality and impact to patient safety of the SaMD, as discussed in IMDRF/SaMD WG/N12.Typically, verification (providing assurance that the design and development activity at each development stage conforms to the requirements) and validation (providing reasonable confidence that the software meets its intended use/user needs and operational requirements) activities ensure that all elements from the SaMD design and development—including any changes made during maintenance/upgrades—have been implemented correctly and that objective evidence of this implementation is recorded.
+
+In short, Barry Boehm expressed the difference between the software verification and software validation as follows:
+
+> Verification: Are we building the product right?
+> Validation: Are we building the right product?
+
+Often these definitions do not match with the definitions used in the medical laboratory. However, in terms of software development and thus bioinformatics, following software verification and software validation definitions make more sense. Noteworthy, the ISO-15189 states that validation is needed for laboratory designed or developed methods and for validated methods subsequently modified. Both are true for software development.
+
+The IEC-62304 mentiones different aspects of verifation; i.e. software unit verification and software integration testing, and validation; i.e. software system test.
+
+## Software verification
+
+Testing whether the code is correctly developed can be done on different levels, for example through:
+
+- Code reviews
+- Unit testing
+- Integration testing
```

Comment:
"can be done on different levels" gives a lot of freedom. I dont really see a minimum verification that will be needed. Under validation (below), I see "the following must be registered", which is very clear, but for this verification part, I would not be sure what is exactly expected of me.

Reply:
> We also use end-to-end workflow tests for software verification. This could be added to this list as long as it stays an example.

added, it already says 'for example'

Reply:
> "can be done on different levels" gives a lot of freedom. I dont really see a minimum verification that will be needed. Under validation (below), I see "the following must be registered", which is very clear, but for this verification part, I would not be sure what is exactly expected of me.

though one ;) added this sentence: "The extend of software verification performed by the Bioinformaticians will differ per team, it should be clear from the procedures which software verification is performed. " is that enough?  Not sure whether we can 'demand' all the steps.